pages/api/update-a-visit.js

```diff
@@ -1,5 +1,28 @@
 import withContainer from "../../src/middleware/withContainer";
 import validateVisit from "../../src/helpers/validateVisit";
+import {
+  NEW_NOTIFICATION,
+  UPDATED_NOTIFICATION,
+} from "../../src/usecases/sendBookingNotification";
+
+const determineNotificationType = (sideA, sideB) => {
```

Comment:
What happens if the original visit used an email address and the updated visit uses a mobile number (and vice versa)? I.e. Key contact is notified via email -> Key contact is notified via text message

What happens if the original visit used an email address and the updated visit adds a mobile number (and vice versa)? I.e. Key contact is notified via email -> Key contact is notified via email and text message

Reply:
Currently, if either the email or phone number change then the new visit notification is sent out on all channels.

Thinking about it, this is less than idea, but not sure what the best approach to take is

Reply:
+1 for only sending the notification to the one that changed (otherwise it may appear like a new visit was booked to whoever already received the notification)

But we can tackle that in a separate PR to keep this one small